submariner/cable-driver-policy.md

```diff
@@ -0,0 +1,250 @@
+
+### Labelling clusters
+
+Clusters are defined as Custom Resources and can be labeled in the CRD. A new parameter to label clusters at
+`subctl join` time could also be added for usability purposes.
+
+### Policy management with subctl
+
+The `CablePolicies` can be created at the Broker deployment through the extension of the Submariner-Operator to
+deploy CablePolicy CRDs, as well as the extension of Submariner configuration YAML to include parameters that
+support cable driver configuration.
+
+> **_NOTE:_** The default policy can be deduced from the existing Submariner configuration if it is not
+explicitly specified.
+
+The `CablePolicies` can be created/managed after Broker deployment through subctl using a command such as:
+
+```cmd
+subctl cable-policy add | list | delete
+```
+
+> **_NOTE:_** Policy CRs are automatically synced to the Broker on creation.
+
+#### subctl cable-policy commands
+
+##### `add`
+
+`subctl cable-policy add [flags]`
+
+<!-- markdownlint-disable line-length -->
+| Flag                                  | Description
+|:--------------------------------------|:--------------------------------------------------------------------------|
+| `--kubeconfig` `<string>`             | Absolute path(s) to the kubeconfig file(s) (default `$HOME/.kube/config`)
+| `--kubecontext` `<string>`            | Kubeconfig context to use
+| `--name` `<string>`                   | The name of this policy (used in the metadata)
+| `--left-cluster-selector` `<string>`  | Comma separated list of cluster labels for the label selector to use
+| `--right-cluster-selector` `<string>` | Comma separated list of cluster labels for the label selector to use
+| `--cable-driver` `<string>`           | The name of the cable driver to use for this policy
+| `--cable-config`  `<string>`          | The cable driver ConfigMap name
+<!-- markdownlint-enable line-length -->
+
+##### `delete`
+
+`subctl cable-policy delete [flags]`
+
+| Flag                       | Description
+|:---------------------------|:-------------------------------------------------------------------------------|
+| `--kubeconfig` `<string>`  | Absolute path(s) to the kubeconfig file(s) (default `$HOME/.kube/config`)
+| `--kubecontext` `<string>` | Kubeconfig context to use
+| `--name` `<string>`        | The name of the policy to delete
+
+##### `list`
+
+`subctl cable-policy list [flags]`
+
+Inspects the cluster and reports information about the detected cable policies.
+
+| Flag                         | Description
+|:-----------------------------|:----------------------------------------------------------------------------|
+| `--kubeconfig` `<string>`    | Absolute path(s) to the kubeconfig file(s) (default `$HOME/.kube/config`)
+| `--kubecontext` `<string>`   | Kubeconfig context to use
+
+#### Connection Policy Examples
+
+- we want to specify a default policy that uses IPsec as the default cable driver.
+
+```yaml
+kind: CablePolicies
+metadata:
+  name: default
```

Comment:
Which namespace does all this go in? (If we require broker access, the broker namespace, but we should mention this either way.)